src/routes/jira/jira-get-connections-backfilStatus.ts

```diff
@@ -0,0 +1,128 @@
+import { NextFunction, Request, Response } from "express";
+import { groupBy, countBy } from "lodash";
+import { RepoSyncState } from "~/src/models/reposyncstate";
+// import { Subscription, TaskStatus } from "~/src/models/subscription";
+import { TaskStatus } from "~/src/models/subscription";
+
+type subscriptionBackfillState = {
+	totalRepos?: number;
+	syncCompleted?: boolean;
+	syncedRepos?: number;
+	backfillSince?: string | null;
+};
+type backFillType = {
+	[key: string]: subscriptionBackfillState;
+};
+
+export const JiraGetConnectionsBackfilStatus = async (
+	req: Request,
+	res: Response,
+	next: NextFunction
+): Promise<void> => {
+	try {
+		const subscriptionIds: number[] = String(req.query.subscriptionIds)
```

Comment:
How can we make sure the passed subscriptionIds belong to the current user? (Aka input validation)?

Reply:
I think we just need to verify the jiraHost in the subscriptions are same as the jiraHost in `res.locals.jiraHost`. Not really mean "belong to current user" :) just current jiraHost.

Reply:
TODO